bukuserver/bookmarklet.js

```diff
@@ -16,4 +16,4 @@ url = "{{url}}" +
     "?url=" + encodeURIComponent(url) +
     "&title=" + encodeURIComponent(title) +
     "&description=" + encodeURIComponent(desc);
-window.open(url, '_blank', 'menubar=no, height=600, width=600, toolbar=no, scrollbars=yes, status=no, dialog=1');
+window.open(url, '_blank', 'menubar=no, height=600, width=800, toolbar=no, scrollbars=yes, status=no, dialog=1');
```

Comment:
600px width is not enough when opening an _existing_ bookmark:
![before](https://github.com/user-attachments/assets/3b177fb4-6b69-4e56-9e65-7a64f799a167 "before")